src/components/project/dto/project.dto.ts

```diff
@@ -49,8 +49,13 @@ class Project extends Resource {
   @Field(() => ProjectStatus)
   readonly status: ProjectStatus;
 
-  @Field()
-  readonly location: SecuredCountry;
+  readonly primaryLocation: Secured<string>;
+
+  readonly nonPrimaryLocation: Secured<string>;
```

Comment:
It doesn't look like this is used since the resolver calls a custom method on the service and doesn't use any properties previously resolved on the Location object from the query.